### PR TITLE
Fixed login wrong username label

### PR DESCRIPTION
**Pull request by bustapaladin**
https://gyazo.com/5d4df37b732aeed846dd4bd653becba9

uname isn't UX friendly. Changed to username as label and placeholder value.

### CODE REVIEW

File: src/components/login.js [hunk 1]
```diff
@@ -169,9 +169,9 @@ class LoginForm extends PureComponent {
 									Uname:
 								</InputGroup.Addon>
 								<input type="text"
-											 placeholder="uname"
+											 placeholder="Username"
 											 className="form-control"
-											 value={this.state.uname}
+											 value={this.state.username}
```
Comment (RHavar) [inline src/components/login.js]: The actual variable needs to still be `uname`

Comment (bustapaladin) [inline src/components/login.js]: oh ok what needs to be done to change the label value then?

Comment (RHavar) [inline src/components/login.js]: Yeah, the placeholder change is correct

Comment (bustapaladin) [inline src/components/login.js]: yea but that will only change the placeholder not the other uname value assigned as the label, EG:

<span class="input-group-addon">Uname:</span>

Comment (RHavar) [inline src/components/login.js]: That's in a different spot, search the code for:
```
								<InputGroup.Addon>
									Uname:
								</InputGroup.Addon>
```

Comment (bustapaladin) [inline src/components/login.js]: Ok should be all good now for merge. 👍  Also submitted another pr but that one needs some testing to see how it displays on diff resolutions. Basically you get the gist of what it tries to do and probably can figure out a better way to do it.

**Comment by RHavar** [comment]
Looks good! Merged it in
